packages/tag/src/index.js

```diff
@@ -0,0 +1,48 @@
+// vendor modules
+import React from 'react';
+import PropTypes from 'prop-types';
+import getClassNames from 'classnames';
+import _ from 'lodash';
+// css modules
+import './index.scss';
+
+/**
+ * @example ../README.md
+ */
+export default class Tag extends React.Component {
```

Comment:
I believe this component also needs rework. It does not satisfy all the variants of tags in the design. Are you expecting depending upon what type we want, we add class and style accordingly? That defeats the whole purpose of global component.

Design: zpl://screen?sid=5e77a90462b4676fb1ec9e7f&pid=5df72d1e602c331cb8dc8b79